src/helm/config/model_metadata.yaml

```diff
@@ -229,6 +229,22 @@ models:
     release_date: 2023-11-21
     tags: [ANTHROPIC_CLAUDE_2_MODEL_TAG, TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG, ABLATION_MODEL_TAG, INSTRUCTION_FOLLOWING_MODEL_TAG]
 
+  - name: anthropic/claude-3-sonnet-20240229
+    display_name: Claude 3 Sonnet (20240229)
+    description: TBD
```

Comment:
Do we have an issue to get these filled?  We need them before we actually deploy...

Reply:
Filed #2435